<?xml version="1.0" encoding="utf-8"?>
<launch>
	
    ### ROOM_CATEGORIZATION ###
    <node name="generate_panoramicImage" pkg="room_categorization" type="generate_panoramicImage.py" output="screen"/>
    <node name="poseRobot" pkg="room_categorization" type="get_poseRobot.py" output="screen"/>
    <node name="objectsInFOV" pkg="room_categorization" type="objectsInFOV.py" output="screen"/>
    <node name="get_bestSubimage" pkg="room_categorization" type="get_bestSubimage.py" output="screen"/>
    <node name="plot_roomScores" pkg="room_categorization" type="plot_roomScores.py" output="screen"/>

	
    ### RVIZ ###
    <node name="rviz_player1" pkg="rviz" type="rviz" args="-d $(find room_categorization)/rviz/room_categorization_rviz_config.rviz"/>

</launch>
